src/v/cluster/tests/feature_barrier_test.cc

```diff
@@ -151,15 +173,18 @@ FIXTURE_TEST(test_barrier_simple, barrier_fixture) {
 
     auto f0 = get_barrier_state(model::node_id{0})
                 .barrier(feature_barrier_tag{"test"});
-    auto f1 = get_barrier_state(model::node_id{1})
+    auto f2 = get_barrier_state(model::node_id{2})
```

Comment:
Why did you switch f1 and f2?

Reply:
probably to match the node id so it's easier to reason bout the assertions below